backoffice/workflows/urls.py

```diff
@@ -1,7 +1,5 @@
-from django.contrib import admin
 from django.urls import include, path
 
 urlpatterns = [
     path("workflows/", include("workflows.urls")),
-    path("admin/", admin.site.urls),
```

Comment:
can we keep both?

Reply:
yep, but not sure if we need both - do we need extra admin url on `backoffice.inspire.net/workflows/backoffice`?

Reply:
I mean, backoffice.inspire.net/dashboard and backoffice.inspire.net/admin